firebase-common/build.gradle.kts

```diff
@@ -20,6 +20,7 @@ android {
     sourceSets {
         getByName("main") {
             manifest.srcFile("src/androidMain/AndroidManifest.xml")
+            res.srcDir("src/androidMain/res")
```

Comment:
can this config be added to test target only?

Reply:
This resource is not required anymore. I've removed it.